zone.go

```diff
@@ -265,24 +265,24 @@ type newZone struct {
 	Type      string `json:"type"`
 	// We use a pointer to get a nil type when the field is empty.
 	// This allows us to completely omit this with json.Marshal().
-	Organization *Organization `json:"organization,omitempty"`
+	Account *Account `json:"organization,omitempty"`
```

Comment:
I've updated the field name to be correct however the JSON response is still for `organizations` so this needs to stay until the API responses are either updated or conditionally respond with different values based on the incoming request.